### PR TITLE
chore: prepare v0.2.21 release

### PR DESCRIPTION
**Pull request by carllerche**

### CODE REVIEW

File: tokio/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+# 0.2.21 (May 12, 2020)
+
+### Fixes
+
+- macros: disambiguate built-in #[test] attribute in macro expansion (#2503)
```
Comment (hawkw) [inline tokio/CHANGELOG.md:5]: nit: should this be
```suggestion
- macros: disambiguate built-in `#[test]` attribute in macro expansion (#2503)
```

**Comment by MikailBag** [comment]
Why `CancellationToken` is not mentioned in changelog?
I think it is very useful feature.

**Comment by carllerche** [comment]
@MikailBag It's flagged as unstable and require opting in with a cfg flag. I am unsure how to document those features. Thoughts @Matthias247 ?

File: tokio/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
+# 0.2.21 (May 12, 2020)
```
Comment (carllerche) [inline tokio/CHANGELOG.md:1]:
```suggestion
# 0.2.21 (May 13, 2020)
```

**Comment by MikailBag** [comment]
Well, I think I should open separate issue for my question about `CancellationToken` stability.

**Pull request merged**